transformer_lens/HookedTransformer.py

```diff
@@ -815,6 +870,7 @@ def from_pretrained(
         n_devices=1,
         tokenizer=None,
         move_to_device=True,
+        padding_side="right",
```

Comment:
This is a rather minor point, but I'm also pondering about which is better between `padding_side` and `default_padding_side`. It's `default_prepend_bos` for `prepend_bos`, but since `padding_side` is an actual attribute of `transformers.Tokenizer`, `padding_side` might be more intuitive for those who know the actual attribute.

Advantages of each:
- `default_padding_side`: It's better for consistency with `default_prepend_bos` and it explicitly tells that it's a default value
- `padding_side`: It's more clear that it will be set as the value of `model.tokenizer.padding_side`